code/framework/object/src/main/java/io/cattle/platform/object/process/ObjectProcessManager.java

```diff
@@ -25,4 +25,8 @@
 
     ExitReason executeProcess(String processName, Object resource, Map<String, Object> data);
 
+    void scheduleProcessInstanceAndPublishEvent(String processName, Object resource, Map<String, Object> data);
```

Comment:
I guess we already use chainProcess term in cattle when we one process is followed by another...while in this case we schedule a single process instance